tasks/sphinx.py

```diff
@@ -196,7 +196,7 @@ def run(self):
                            FIRST(c.type),
                            FIRST(c.extra),
                            FIRST(c.aggregate),
-                           JSONB_Object_Agg(t.id, t.name),
+                           JSONB_Object_Agg(t.type || '/' || t.id, t.name),
```

Comment:
Handled in https://github.com/CartoDB/bigmetadata/pull/164, so we don't need to merge to get this fix.